tools/bamm-cli/src/test/java/io/openmanufacturing/sds/ApplicationIntegrationTest.java

```diff
@@ -310,4 +311,96 @@ private void createValidArgsExecution( final KnownVersion testedVersion, final F
       argumentsList.addAll( Arrays.asList( args ) );
       return argumentsList;
    }
+
+   @ParameterizedTest
+   @MethodSource( value = "allVersions" )
+   public void testGenerateJavaClassWithoutCopyright( final KnownVersion metaModelVersion ) throws Throwable {
+      final File outputDir = outputDirectory.toFile();
+
+      createValidArgsExecution( metaModelVersion, outputDir, "-java" );
+      final byte[] testAspectRaw = Files
+            .readAllBytes( outputDir.toPath().resolve( "io/openmanufacturing/test/AspectWithEntity.java" ) );
+      final String testAspect = new String( testAspectRaw );
+      assertThat( testAspect ).doesNotContain( "Copyright" );
+
+      final byte[] testEntityRaw = Files
+            .readAllBytes( outputDir.toPath().resolve( "io/openmanufacturing/test/TestEntity.java" ) );
+      final String testEntity = new String( testEntityRaw );
+      assertThat( testEntity ).doesNotContain( "Copyright" );
+   }
+
+   @ParameterizedTest
+   @MethodSource( value = "allVersions" )
+   public void testGenerateJavaClassWithCopyright( final KnownVersion metaModelVersion ) throws Throwable {
+      final File outputDir = outputDirectory.toFile();
+      final String currentWorkingDirectory = System.getProperty( "user.dir" );
+      final String templateLibPath = currentWorkingDirectory + "/../../core/sds-aspect-model-java-generator/templates";
+      final int currentYear = LocalDate.now().getYear();
+      final String expectedCopyright = String.format( "Copyright (c) %s OMP Test GmbH. All rights reserved", currentYear );
```

Comment:
I'd change "GmbH" to "Inc." or similar in order to remove unnecessary Germany-centricity.